SUMMARY.md

```diff
@@ -5,49 +5,54 @@
 * [Status](status.md)
 * [Contributing](contributing.md)
 * [Drupal 8 Core](core/file-systems.md)
-   * [File Systems](core/file-systems.md)
-   * [File and Image Fields](core/file_and_image.md)
-   * [WYSIWYG](core/wysiwyg.md)
-   * [Limitations](core/limitations.md)
+    * [File and Image Fields](core/file_and_image.md)
+    * [WYSIWYG](core/wysiwyg.md)
+    * [Limitations](core/limitations.md)
 * [Browsing](browsing.md)
 * [Embedding](embedding.md)
 * [Modules](modules/intro.md)
-   * [Entity Browser](modules/entity_browser/intro.md)
-       * [Architecture](modules/entity_browser/architecture.md)
-       * [Displays](modules/entity_browser/displays.md)
-       * [Widgets](modules/entity_browser/widgets.md)
-       * [Widget Selectors](modules/entity_browser/widget_selectors.md)
-       * [Selection displays](modules/entity_browser/selection_displays.md)
-       * [Creating a browser trough the UI](modules/entity_browser/creating_browser_through_ui.md)
-       * [Configuration file structure](modules/entity_browser/configuration.md)
-       * [Inline entity form integration](modules/entity_browser/inline_entity_form.md)
-   * [DropzoneJS](modules/dropzonejs/intro.md)
-   * [Embed API](modules/embed/intro.md)
-   * [Entity Embed](modules/entity_embed/README.md)
-       * [Button configuration](entity_embed/button.md)
-       * [WYSIWYG configuration](entity_embed/formats.md)
-       * [Embedding an entity using WYSIWYG](entity_embed/wysiwyg.md)
-       * [`<drupal-entity>` HTML tag specification](entity_embed/tag_specification.md)
-       * [Extending](entity_embed/INTEGRATION.md)
-           * [Events and hooks](entity_embed/events_hooks.md)
-           * [Custom data attributes](entity_embed/data_attributes.md)
-           * [Display plugins](entity_embed/display_plugins.md)
-   * [URL Embed](modules/url_embed/intro.md)
-   * [File Entity](modules/file_entity/intro.md)
-   * [File Entity Browser](modules/file_browser/README.md)
-   * [Media Entity](modules/media_entity/intro.md)
-       * [Image](modules/media_entity/image.md)
-       * [Audio](modules/media_entity/audio.md)
-       * [Slideshow](modules/media_entity/slideshow.md)
-       * [Embedable video](modules/media_entity/embeddable_video.md)
-       * [Twitter](modules/media_entity/twitter.md)
-       * [Instagram](modules/media_entity/instagram.md)
-   * [Crop API](modules/crop/intro.md)
-       * [Architecture](modules/crop/architecture.md)
-       * [Implementation examples](modules/crop/Implementation_examples.md)
-   * [ImageWidgetCrop](modules/image_widget_crop/intro.md)
-   * [Fallback Formatter]((modules/fallback_formatter/intro.md))
-   * [Field formatter](modules/field_formatter/intro.md)
-   * [Flysystem](modules/flysystem/intro.md)
-   * [Image Effects](modules/image_effects/intro.md)
+    * [Entity Browser](modules/entity_browser/intro.md)
```

Comment:
These diffs seem unrelated, could we leave in the existing spacing and fix it in a follow-up PR?